tests/factories/filterObjectFactory.js

```diff
@@ -0,0 +1,121 @@
+import faker from "faker";
+
+import { Factory } from "./factory";
+
+export class VersionFilterObjectFactory extends Factory {
+  getFields() {
+    return {
+      type: "version",
+      versions: [],
+    };
+  }
+
+  postGeneration() {
+    const versionSet = new Set();
+    const { generateVersionsCount } = this.options;
+    for (let i = 0; i < generateVersionsCount; i++) {
+      versionSet.add(faker.random.number({ min: 40, max: 100 }));
+    }
```

Comment:
nit: its possible if you set `generateVersionsCount` to `n` that you will actually get `< n` values due to de-duplication. a better approach is probably to use a `while`:
```js
const { generateVersionsCount = 0 } = this.options;
while (versionSet.size < generateVersionsCount) {
  versionSet.add(fake.random.number({ min: 40, max: 100 });
}
```